src/python/pants/backend/python/target_types.py

```diff
@@ -307,7 +335,7 @@ def determine_packages_to_cover(
 class PythonTestsTimeout(IntField):
     """A timeout (in seconds) which covers the total runtime of all tests in this target.
 
-    This only applies if `--pytest-timeouts` is set to True.
+    This only applies if the option `--pytest-timeouts` is set to True.
```

Comment:
I used the command line form because we expect users to keep this default on and only disable it temporarily via the command line. See the tip block in https://pants.readme.io/docs/python-test-goal#using-timeouts.